database/init.sql

```diff
@@ -1,67 +1,99 @@
 BEGIN;
 
-DROP TABLE IF EXISTS employee, role, application, site, country, project, iteration, milestone, scrum CASCADE;
+DROP TABLE IF EXISTS job, application, site, project, iteration, milestone, scrum, employee, 
+employee_scrum, absence, employee_absence, users CASCADE;
+DROP TYPE role_access_tier CASCADE;
```

Comment:
This is missing "if exists" cause when I tested it, it gave me the following error
`error: type "role_access_tier" does not exist`


```suggestion
DROP TYPE IF EXISTS role_access_tier CASCADE;
```